client_test.go

```diff
@@ -142,7 +142,7 @@ func TestUpdateFeatureState(t *testing.T) {
 
 	client := flagsmithapi.NewClient(MasterAPIKey, server.URL+"/api/v1")
 
-	updated_fs, err := client.UpdateFeatureState(&fs)
+	err := client.UpdateFeatureState(&fs, false)
```

Comment:
Is there a way to have default values or overloads in Go so we don't have to provide the `updateSegmentPriority` boolean unless needed?